backend/fetchArrivals.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+const key = import.meta.env.VITE_API_KEY;
+
+export default async function getArrivals(id: string){
+    let url = `https://api.tfl.gov.uk/StopPoint/${id}/Arrivals`;
+   
+    if (key !== ""){
+        url = url + `?=${key}`;
```

Comment:
You haven't included the param name for the API key - see [here](https://techforum.tfl.gov.uk/t/basic-documentation/3072/2):

> When making an API request, please append the app_key as a query parameter to your requests. Please ignore any references to passing the app_id as this is no longer required.

And example from [here](https://techforum.tfl.gov.uk/t/basic-documentation/3072/2):

> https://api.tfl.gov.uk/Mode/tube/Arrivals?app_key=YOUR_APP_KEY